include/seqan3/range/view/get.hpp

```diff
@@ -64,7 +64,12 @@ inline auto const get = view::transform([] (auto && in) -> decltype(auto)
     using seqan3::get;
     static_assert(tuple_like_concept<decltype(in)>,
                   "You may only pass ranges to view::get whose reference_t models the tuple_like_concept.");
-    return get<index>(std::forward<decltype(in)>(in));
+
+    // we need to explicitly remove && around temporaries to return values as values (and not as rvalue references)
+    // we cannot simply cast to std::tuple_element_t (or set that as return value), because some tuples, like
+    // our cartesian_composition alphabets do not return that type when get is called on them (they return a proxy)
```

Comment:
We could discuss at the strategy meeting if this is the expected behaviour or if the tuple_element_type should actually be the proxy